### PR TITLE
Add JupyterWidget Project for HCK06

### PR DESCRIPTION
**Pull request by thewtex**
@oruebel @mgrauer @bendichter @jcfr please take a look.

### CODE REVIEW

**Comment by oruebel** [comment]
Please also update https://github.com/NeurodataWithoutBorders/nwb_hackathons/blob/master/HCK06_2019_Janelia/projects/PROJECTS.md to add the project to the list of assigned projects.

**Comment by thewtex** [comment]
> Please also update https://github.com/NeurodataWithoutBorders/nwb_hackathons/blob/master/HCK06_2019_Janelia/projects/PROJECTS.md to add the project to the list of assigned projects.

Done!

**Comment by oruebel** [comment]
It would be interesting to expand the project to also have a component for planing of more general visualization needs. The ECoG surface is interesting because it allows to show some nice visualization and is something that seems simple enough to complete at the hackathon. However, it is (at least right now) also a bit specific since it relies on an extension from one lab. Some more general other visualizations that I think would be broadly useful would be to support: i) visualizations of dynamic tables as table views, ii) support visualization of time intervals tables (including resolution of links to timeseries) to be able to show the data the time interval annotations apply to, iii) support visualization of ophys image series and ROIs etc. I think some of these should be pretty simple to do (and maybe you already support some of this).

File: HCK06_2019_Janelia/projects/PROJECTS.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 The lists below provide an overview of specific projects for which at least one or more attendees have signed up for to contribute to. Projects are collaborative in nature. If you are attending the hackathon and are interested in participating in any of the existing projets then please contact the team listed on the project page. 
 
+* **Python/PyNWB**
+
+   * **Visualization and documentation tools:**
```
Comment (oruebel) [inline HCK06_2019_Janelia/projects/PROJECTS.md]: Can you please move the ``[**NWB:N Jupyter notebook widgets**](projects/JupyterWidgets)`` to the ```###Developer Days``` list

**Comment by bendichter** [comment]
@oruebel 

Dynamic tables are already rendered as table views using pandas. This includes intervals (although the links to TimeSeries are not resolved). We have not looked much into ophys datasets like ROIs, but I agree that would be a good direction.

**Comment by oruebel** [comment]
> Dynamic tables are already rendered ... 

That's great. Having a feature to be able to follow references, e.g,. from an ElectricalSeries to the electrodes-table or from an TimeIntervals table to the TimeSeries would be a cool feature.

I think having the jupyter widgets is great. I'm looking forward to chatting with you guys more about this at the hackathon.

File: HCK06_2019_Janelia/projects/PROJECTS.md (hunk 1 above)
Comment (thewtex) [reply HCK06_2019_Janelia/projects/PROJECTS.md]: Yes -- done!

**Comment by thewtex** [comment]
> I think having the jupyter widgets is great. I'm looking forward to chatting with you guys more about this at the hackathon.

Ditto!

**Pull request merged**